resources/metadata/sonarExecuteScan.yaml

```diff
@@ -53,6 +53,8 @@ spec:
             default: sonar
           - name: sonarTokenCredentialsId
             type: secret
+          - name: sonar
+            type: trustengineSecret
```

Comment:
We should probably do this in a separate PR, so that the new functionality doesn't get activated yet.